templates/webserver/webserver-deployment.yaml

```
@@ -76,6 +76,39 @@ spec:
           env:
           {{- include "standard_airflow_environment" . | indent 10 }}
       containers:
+        {{- if .Values.authSidecar.enabled }}
+        - name: auth-proxy
+          image: "{{ .Values.authSidecar.repository }}:{{ .Values.authSidecar.tag }}"
+          imagePullPolicy: {{ .Values.authSidecar.pullPolicy }}
+          ports:
+          - containerPort: {{ .Values.authSidecar.authproxyPort }}
```

```suggestion
          - containerPort: {{ .Values.authSidecar.port }}
```

This would be my vote 🤷‍♂️